tracee/tracee.bpf.c

```diff
@@ -497,14 +500,39 @@ static __always_inline int get_config(u32 key)
     return *config;
 }
 
+// returns 1 if you should trace based on uid, 0 if not
 static __always_inline int uid_filter_matches()
 {
+
+    int equal;
+    int not_equal;
+
     uid_t uid = (u32)bpf_get_current_uid_gid();
-    u32* isMatch = bpf_map_lookup_elem(&uid_filter, &uid);
-    if (isMatch == NULL) {
-        return 0;
+    int64_t* equality = bpf_map_lookup_elem(&uid_equal_filter, &uid);
+    if (equality == NULL) {
+        equal = 0;
+        not_equal = 0;
+    } else {
+        not_equal = *equality == 0;
+        equal = *equality == 1;
     }
-    return 1;
+
+    u32 greater = ZERO;
+    u32 less = ONE;
```

Comment:
Was this what you meant by using `#define` constants?

Reply:
I would give it a more meaningful name, such as GREATER and LESS.
The point is to use the same names (but in camel case) in the go code, as the maps are shared